nodejs/week1/homework/src/backend/app.js

```diff
@@ -0,0 +1,92 @@
+const express = require("express");
+const app = express();
+
+
+// import data here
+const meals = require("./data/meals.json");
+const reservations = require("./data/reservations.json");
+const reviews = require("./data/reviews");
+//console.log(meals[0].title);
+
+// this is where you will be adding your routes
+app.get("/", async (request, response) => {
+  response.send("asd");
+});
+
+/*
+Respond with the json for all the meals. For each meal, if there are reviews matching it's meal ID, add these reviews to each meal in the form of an array. For meals that do not have any reviews, the "reviews" property will be an empty array. (watch the GIF below to understand how it should be structured)
+*/
+
+app.get("/meals", async(request, response)=>{
+  const allTheMeals = meals.map(meal=>{
+    const reviewOfEachMeal = reviews.filter(review=>review.mealId===meal.id)
```

Comment:
Nitpick(readability):
Code becomes much easier to read when you add spaces between operators, this applies for many places in your code.

Like this:
`reviews.filter(review => review.mealId === meal.id)`